ports/libsimpleservo/capi/src/vslogger.rs

```diff
@@ -12,7 +18,10 @@ pub struct VSLogger;
 
 impl log::Log for VSLogger {
     fn enabled(&self, metadata: &Metadata) -> bool {
-        metadata.level() <= Level::Warn
+        let _modules = LOG_MODULE_FILTERS.lock().unwrap();
+        let is_module_enabled =
+            _modules.contains(&String::from(metadata.target())) || _modules.is_empty();
```

Comment:
The `&String::from` should not be necessary.

Reply:
I had it without the &String::from() and got an error.  Here is the error I get by removing.

C:\git-mozilla\servo-angelortiz1007>mach build -d --uwp
   Compiling simpleservo_capi v0.0.1 (C:\git-mozilla\servo-angelortiz1007\ports\libsimpleservo\capi)
error[E0308]: mismatched types
  --> ports\libsimpleservo\capi\src\vslogger.rs:21:50
   |
21 |         let is_module_enabled = modules.contains(metadata.target()) || modules.is_empty();
   |                                                  ^^^^^^^^^^^^^^^^^ expected struct `std::string::String`, found str
   |
   = note: expected type `&std::string::String`
              found type `&str`

error: aborting due to previous error

Reply:
Ah, I understand why that's necessary now. We can ignore my request, in that case.